themes/base.theme.bash

```diff
@@ -464,11 +464,13 @@ function aws_profile {
 }
 
 function safe_append_prompt_command {
-    if [[ -n $1 ]] ; then
-        case $PROMPT_COMMAND in
-            *$1*) ;;
-            "") PROMPT_COMMAND="$1";;
-            *) PROMPT_COMMAND="$1;$PROMPT_COMMAND";;
-        esac
+    local prompt_re="\<${1}\>" # exact match regex
```

Comment:
This will work on Linux, but not on BSD-style operating systems like macOS.

This check fails for me on macOS Sierra. According to http://stackoverflow.com/q/9792702/1228454 the `\<` pattern for word boundaries only works on Linux, and for BSD-liks OSes, you have to use `[[:<:]]` (and `[[:>:]]` instead.

Can you please add a check for the OS, and then depending on the OS, you set the `prompt_re` variable? The rest of the code should work in both cases...